waku/v2/protocol/waku_message/message.nim

```diff
@@ -12,7 +12,10 @@ else:
 import
   ../../utils/time
 
-const MaxWakuMessageSize* = 1024 * 1024 # In bytes. Corresponds to PubSub default
+const
+  MaxTopicAttributeLength* = 128 # 128 bytes
```

Comment:
perhaps nitpick but can we add "content topic" to it? to differenciate from the pubsub?